horace_core/algorithms/write_nsqw_to_sqw.m

```diff
@@ -152,9 +152,6 @@
 [header_combined,nspe] = sqw_header.header_combine(header,allow_equal_headers,drop_subzone_headers);
 
 
-if numel(datahdr) > 1
```

Comment:
what is the consequence of this check being dropped?

Reply:
I've found this exactly check sitting as last row of `header_combine`, so its just dropped duplicaton